client/src/components/NavBar/NavBar.tsx

```diff
@@ -94,6 +94,11 @@ export function Navbar(props: any) {
       <Link to="review">Review</Link>
       <Link to="reviews">Reviews</Link>
       <StyledButton onClick={() => testAPICalls()}>Test API</StyledButton>
+
+      <Link to={"/single-result/ChIJn58N1B9gUocRpAXOXPbFcOo"}>
+        Single Result
+      </Link>
```

Comment:
Not sure what this is for, don't we need a link here for the review stuff that you built in this PR?